reference/Upload-API.yaml

```diff
@@ -879,5 +879,299 @@ paths:
                       etag: 55a64df551425fcc55e4d42a148795d9f25f89d6
         description: The request body should contain a reference to each part along with the Etag of each upload.
     parameters: []
+  '/api/v1/{apiUserId}/{apiKey}/{targetId}/{languageCode}/search':
+    parameters:
+      - schema:
+          type: string
+        name: apiUserId
+        in: path
+        required: true
+        description: Guid identifying the requesting client
+      - schema:
+          type: string
+        name: apiKey
+        in: path
+        required: true
+        description: Second layer encrypted identifier for the requesting client
+      - schema:
+          type: string
+        name: targetId
+        in: path
+        description: Guid identifying the target where articles are allowed to be published
+      - schema:
+          type: string
+        name: languageCode
+        in: path
+        required: true
+        description: Two characters ISO language code
+    get:
+      summary: Article search
+      tags: []
+      responses:
+        '200':
+          description: |-
+            A sample response
```

Comment:
The response does not have a fixed but dynamic one, Only a part of the response is fixed

Reply:
Done

Reply:
hmm, OK, trying to understand how metadata cannot be fixed 🤔 
there may be lots of possibilities, but can it have a defined structure, for example:
```
...
metadata: {
    id: '12345678abcd'
    name: 'favouritePlayer'
    value: 'FooBarBaz'
}
...
```

Reply:
No, it can't have a definite structure. Fields are dynamically created and are of different types. Few fields however are compulsory, they are:
1. body
2. title
3. videoduration

all other fields are optional and dynamic